### PR TITLE
DEVDOC-145: Add tips about shared er schema

### PR DESCRIPTION
**Pull request by flower-of-the-bridges**
## Description

The real time updater and single view trigger configuration pages now contains a tip panel to show how er schema config maps can be shared

## Pull Request Type

- [x] Documentation content changes
- [ ] Bugfix / Missing Redirects
- [ ] Docusaurus site code changes

## PR Checklist

- [x] The commit message follows our guidelines included in the [CONTRIBUTING.md](../blob/main/CONTRIBUTING.md#how-to-submit-a-pr)
- [x] All tests of Lint, cspell and check-content pass. ([How to launch them?](../blob/main/CONTRIBUTING.md#content-checks))
- [x] No sensitive content has been committed

### CODE REVIEW

File: docs/fast_data/configuration/realtime_updater.md [hunk 1]
```diff
@@ -239,6 +253,15 @@ The ER Schema ConfigMap is created after the service is attached to a System of
 This is an empty configuration: the Real-Time Updater Microservice could be deployed without pod restart, but this file must be modified according to the projections associated with this microservice to work properly.
 :::  
 
+:::tip Shared ER Schema
+
+If you have already [attached an ER Schema to a Single View Creator](/fast_data/configuration/single_view_creator.md#selecting-an-er-schema-with-the-no-code), you can re-use it in the Real-Time Updater configuration too.
+
+By deleting the default one and [add an existing Config Map](/development_suite/api-console/api-design/services.md#shared-configmaps) corresponding to the one of the Er Schema, changes made to an Er Schema will be reflected also in the Real-Time Updater.
```
Comment (danibix95) [inline docs/fast_data/configuration/realtime_updater.md:260]: If possible, I would also state that it is not recommended to change the ER schema from micro-services section. Instead users should rather exploit the dedicated section in Fast Data configurator.

Comment (flower-of-the-bridges) [reply docs/fast_data/configuration/realtime_updater.md:260]: Sure, I can put a warning panel

Comment (akelity) [reply docs/fast_data/configuration/realtime_updater.md]: @danibix95 and @TheRusCompass any other comment on that?
We want to merge this PR asap (if possible)

**Review by TheRusCompass — APPROVED** [review]
LGTM

**Pull request merged**